test/index.js

```diff
@@ -0,0 +1,14 @@
+import assert     from 'assert'
+import superagent from 'superagent'
+
+require('../lib/index')
+
+describe('/version', () => {
+  it('returns the current version', (done) => {
+    superagent.get('http://localhost:8000/version')
```

Comment:
(maybe that's not a clear answer: yes, the server needs to be running for this to work. I'm not sure how else to test an API like this)